test/WebSites/RazorPagesWebSite/Pages/ViewData/ViewDataInPage.cs

```diff
@@ -0,0 +1,32 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace RazorPagesWebSite
+{
+    public class ViewDataInPage : PageModel
+    {
+        [ViewData]
+        public string Title => "Title with default value";
+
+        [ViewData]
+        public string Keywords { get; set; }
+
+        [ViewData]
+        public  string Description { get; set;}
+
+        [ViewData(Key = "Author")]
+        public string AuthorName { get; set; }
+
```

Comment:
Can one of these be a `TempData` property? I see unit tests that have both types of properties but not a functional test.

Reply:
We have independent tests for TempData. I'd rather not make this thing combinatorial especially given that we are expected to do `CookieData` and `SessionData`